src/commands/appSettings/AzureWebJobsStorageExecuteStep.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep, IStorageAccountWizardContext } from 'vscode-azureextensionui';
+import { localEmulatorConnectionString } from '../../constants';
+import { azureWebJobsStorageKey, setLocalAppSetting } from '../../funcConfig/local.settings';
+import { getStorageConnectionString } from '../../utils/azure';
+import { IAzureWebJobsStorageWizardContext } from './IAzureWebJobsStorageWizardContext';
+
+export class AzureWebJobsStorageExecuteStep<T extends IAzureWebJobsStorageWizardContext> extends AzureWizardExecuteStep<T> {
+    public priority: number = 230;
```

Comment:
Where is this number coming from?  Is 230 arbitrary?

Reply:
All execute steps need a priority to say when they execute
https://github.com/Microsoft/vscode-azuretools/pull/463
https://github.com/Microsoft/vscode-azuretools/pull/464
https://github.com/Microsoft/vscode-azurefunctions/pull/1143